tsconfig.json

```diff
@@ -1,21 +1,30 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "lib": ["dom", "dom.iterable", "esnext"],
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
     "declaration": true,
     "allowJs": true,
     "skipLibCheck": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
-    "module": "es6",
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
+    "outDir": "./.webpack/main",
+    "module": "commonjs",
```

Comment:
You cannot change module resolution just like that - there should be solid reason to do that. Ask Nartay - he did some thorough research on module resolution system and may help you with that. But as far as I am concerned you shouldn't change it